opte-api/src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub use ulp::*;
 ///
 /// We rely on CI and the check-api-version.sh script to verify that
 /// this number is incremented anytime the oxide-api code changes.
-pub const API_VERSION: u64 = 19;
```

Comment:
Apologies for the delay here. This change looks good, though I don't see why the API version needs to be bumped.

Reply:
yup, don't need this anymore, removed it